src/packages/@ncigdc/components/PortalContainer.js

```diff
@@ -62,10 +57,8 @@ const PortalContainer = ({
       id="skip"
       role="main"
       style={{
+        minWidth: 1024,
         paddingBottom: '120px',
-        paddingTop: `calc(51px + ${notifications.filter(n => !n.dismissed)
```

Comment:
Instead of having the padding be calculated manually with a fixed multiplier, we let the browser carry the weight using css flow.